dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -60,6 +63,18 @@ def __init__(self, name: str | None, options: CategoricalOptions = None) -> None
             self.max_sample_size_to_check_stop_condition = (
                 options.max_sample_size_to_check_stop_condition
             )
+            self._cms_confidence = options.cms_confidence
+            self._cms_relative_error = options.cms_relative_error
+
+            if options.cms:
+                self._heavy_hitters_threshold = options.heavy_hitters_threshold
+                self.num_hashes = count_min_sketch.suggest_num_hashes(
+                    options.cms_confidence
+                )
+                self.num_buckets = count_min_sketch.suggest_num_buckets(
+                    options.cms_relative_error
+                )
+                self.cms = count_min_sketch(self.num_hashes, self.num_buckets)
```

Comment:
@JGSweets having second thoughts that cms needs to be defined here... we only ever use it within _update_categories(), where we get an heavy-hitter approximate category count and use that to update self._categories. I think as long as self._categories persists we are safe and don't need to also persist cms

...Also having self.cms defined here causes serialization troubles. count_min_sketch has its own serialization method so we'd need some custom workaround to get json.dumps() to work with it out of the box if we choose to keep it defined here...